charts/aws-calico/README.md

```diff
@@ -1,3 +1,13 @@
+# Important Announcement: AWS EKS will no longer maintain and update Calico charts in this repository #
+- The current Calico charts will not be updated
+- The current Calico charts will be removed from this repository on March 31, 2023
+- Current open requests will be redirected to Calico
+- You should submit new requests to Calico directly
+    - For Calico, please send issues to [Calico repository](http://github.com/projectcalico/calico)
+    - For Tigera Operator, please send issues to [Operator repository](http://github.com/tigera/operator)
+- We will update and maintain the [AWS docs](https://docs.aws.amazon.com/eks/latest/userguide/calico.html) for Calico installation after the deprecation
```

Comment:
I am thinking we can replace the below instructions to match the AWS docs instructions. Any thoughts?

Reply:
Considering this README serves helm installation from EKS repo by "historical" way and AWS Doc serves instructing users to install Calico from Tigera helm repo, we may want to keep below instructions till we remove charts and the old instructions? Basically there will be no Calico charts change in this repo. Probably instructions should stay same?